.vscode/launch.json

```diff
@@ -81,6 +81,26 @@
         "order": 2
       }
     },
+    {
+      "name": "Debug Extension & Build Server",
+      "type": "extensionHost",
+      "request": "launch",
+      "runtimeExecutable": "${execPath}",
+      "args": [
+        "--extensionDevelopmentPath=${workspaceFolder}/extension"
+      ],
+      "outFiles": [
+        "${workspaceFolder}/extension/dist/**/*.js"
+      ],
+      "preLaunchTask": "Gradle: Build",
+      "presentation": {
+        "group": "debug",
+        "order": 3
+      },
+      "env": {
+        "DEBUG_VSCODE_BUILD_SERVER":"true"
```

Comment:
`DEBUG_VSCODE_BUILD_SERVER` -> `DEBUG_GRADLE_BUILD_SERVER`

(and for all other occurrence)